recipes/re2/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+set -x
+
+builddir="${SRC_DIR}/cmake-build"
+mkdir -p ${builddir}
+cd ${builddir}
+cmake -DBUILD_SHARED_LIBS=ON \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -DCMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}" \
+      "${SRC_DIR}"
+make -j "${CPU_COUNT}"
+make -j "${CPU_COUNT}" test
```

Comment:
ctest can be used to run tests and it can skip tests.
`ctest -E "dfa|exhaustive|random"` would skip the tests in your patch and you don't need the patch